src/client.ts

```diff
@@ -622,6 +623,10 @@ export class GenesysCloudWebrtcSdk extends (EventEmitter as { new(): StrictEvent
     return this.sessionManager.endSession(endOptions);
   }
 
+  async forceTerminateSession (sessionId: string, reason?: Constants.JingleReasonCondition): Promise<void> {
```

Comment:
It may be good to add the docs here as a tsdoc. That helps out when looking through functions in the text editor.